configuration.go

```diff
@@ -0,0 +1,71 @@
+package corbado
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/corbado/corbado-go/pkg/assert"
+	"github.com/corbado/corbado-go/pkg/generated/api"
+)
+
+type Configuration struct {
+	ProjectID              string
+	APISecret              string
+	FrontendAPI            string
+	BackendAPI             string
+	ShortSessionCookieName string
+	CacheMaxAge            time.Duration
+	JWTIssuer              string
+
+	JWKSRefreshInterval  time.Duration
+	JWKSRefreshRateLimit time.Duration
+	JWKSRefreshTimeout   time.Duration
+
+	HTTPClient         *http.Client
+	ExtraClientOptions []api.ClientOption
+}
+
+const (
+	configurationDefaultBackendAPI             string = "https://backendapi.corbado.io"
+	configurationDefaultFrontendAPI            string = "https://%s.frontendapi.corbado.io"
```

Comment:
we should probably add the logic that checks if project config environment == prod and then forces this URL to be redefined. So that we don't bump into issues with cross-origin cookies like we did recently

Reply:
I added this to the backendapi so we have this check for every SDK :)